.github/workflows/php.yml

```diff
@@ -0,0 +1,52 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ 2.x ]
+  pull_request:
+    branches: [ 2.x ]
```

Comment:
Scheduled daily builds could be also useful to see if a change in upstream causes breaking change here.